### DISCUSSION TITLE
ssl نگرفتن

### DISCUSSION THREAD

**Discussion by Faramarzbal** [hiddify/Hiddify-Manager#1063] [Bug باگ: please use issue]
چند روز پییش بعد این که تو یک روز چندین بار واس دامنه اصلی ssl گرفتم یعنی پنلو هی پاک کردمو مجدد نصب کردم دیگه موفق نشدم با دامنه اصلی ssl بگیرم. با سابدامینای همون دامنه بدون ایراد بالا میاد ولی اگه از دامنه اصلیم استفاده کنم پنل با دامنه اصلی بدون ssl بالا میاد . بنظرتون مشکل کجاست ؟ درضمن همه چی اوکیه یعنی ایپی روی داممنه تو dns ست میشه. چند روزیم با دامنه اصلی ssl نگرفتم .

**Comment by Faramarzbal** [comment]
بنظرتون مشکل از پنله یا cloudflare?

**Comment by adljooy** [comment]
ممکنه تو .htaccess ریدایرکتی چیزی داشته باشین.
این کد رو اضافه کنین بهش :

RewriteEngine On
RewriteCond %{HTTPS} off
RewriteRule ^(.*)$ https://%{HTTP_HOST}%{REQUEST_URI} [L,R=301]
